client/state/data-layer/wpcom/sites/comments/likes/new/index.js

```diff
@@ -28,15 +34,27 @@ export const likeComment = ( { dispatch }, action ) => {
 	);
 };
 
-export const updateCommentLikes = ( { dispatch }, { siteId, postId, commentId }, { like_count } ) =>
+export const updateCommentLikes = (
+	{ dispatch },
+	{ siteId, postId, commentId, status },
+	{ like_count }
+) =>
 	dispatch(
-		bypassDataLayer( {
-			type: COMMENTS_LIKE,
-			siteId,
-			postId,
-			commentId,
-			like_count,
-		} )
+		withAnalytics(
+			composeAnalytics(
+				recordTracksEvent( 'calypso_comment_management_like', {
+					also_approve: 'unapproved' === status,
+				} ),
+				bumpStat( 'calypso_comment_management', 'comment_liked' )
```

Comment:
We will be liking things in other areas like Notifications/Reader etc, so we won't be able to such a specific event like `calypso_comment_management_like` or `calypso_comment_management_like`. An alternative might be to have a generic `calypso_comment_like`, with a source prop that can be set with some other event meta?

cc @dmsnell, thoughts?

Reply:
The other stat grouping of `calypso_comment_management` I think is useful, and would probably remain in the calling component?

Reply:
not sure I follow, but I do make a distinction between things we intrinsically want to track (liking a comment) and _campaign_-like things we want to track (this particular button was clicked, a comment was liked _here_)

so I think it's fine to wrap certain actions in stat-tracking meta from inside the UI components if that's where our intentions to track them are in real life.

Reply:
> An alternative might be to have a generic `calypso_comment_like`, with a source prop that can be set with some other event meta?

This sounds good to me. As long as we can filter generic `calypso_comment_like` events by source that they are coming from in Tracks we should be fine.